policyengine_us/variables/gov/states/hi/tax/income/deductions/itemized/hi_reduced_itemized_deduction.py

```diff
@@ -0,0 +1,63 @@
+from policyengine_us.model_api import *
+
+
+class hi_reduced_itemized_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii reduced itemized deduction"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=15"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=32"  # total itemized deduction worksheet
+    )
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.deductions.itemized
+        p_deductions = parameters(period).gov.irs.deductions
+
+        total_deductions = tax_unit("hi_total_itemized_deduction", period)
+
+        medical_expense = add(tax_unit, period, ["medical_expense"])
```

Comment:
We should make hi_medical_expense_deduction file and others if we are using the calculation multiple times